### ISSUE TITLE
exclude by route

### ISSUE THREAD

**Issue by japel** [villadora/express-bunyan-logger#36]
Is it possible to exclude some routes from being logged?
If so, how can this be done?

**Comment by marcbachmann** [comment]
Only mount the middleware on the routes where you need it.
We shouldn't do this in that library.

**Comment by advance512** [comment]
I disagree. Generally I want *every* route to be logged, by default. I don't want to "opt in", I want to "opt out". e.g. /ping from ELB.

Any way to do this, currently?
